utils/useProcessGame.tsx

```diff
@@ -0,0 +1,59 @@
+import { useCallback } from 'react';
+import { getGameData, getUserPick } from './utils';
+import { useDataStore } from '@/store/dataStore';
+
+interface UseProcessGameProps {
+  leagueId: string;
+  gameWeek: { id: string };
+  user: { id: string };
+  NFLTeams: any; // Define this type based on your actual data structure
```

Comment:
Isn't this just `INFLteams`? Also, why didn't ES Lint complain about this file? There's no copyright, and `any` is not accepted as a type.

Reply:
not sure why! but it is now fixed as it screamed about it when I opened the file ... I did run pnpm run build to see any warnings / errors.